financial-connections/src/test/java/com/stripe/android/financialconnections/FinancialConnectionsSheetViewModelTest.kt

```diff
@@ -281,7 +281,7 @@ class FinancialConnectionsSheetViewModelTest {
             viewModel.handleOnNewIntent(errorIntent)
 
             // Then
-            withState(viewModel) {
+            viewModel.stateFlow.value.let {
```

Comment:
Maybe we can add some `FinancialConnectionsViewModel` specific test extensions like Mavericks had. What do you think?

Reply:
good idea! noted for future PRs : )